### PR TITLE
Adjusted hospital_3 - Georgetown

### LINKED ISSUE #44: Tanoa Hospitals

### PR DESCRIPTION
**Pull request by Heavybob**
Resolves #44<issue ID here>. 

#### Changes proposed in this pull request: 
- Adjusted hospital_3 to be at a better location. Further tweaking will be required to remove the giant building which spawns on the hospital_3 marker but apart from that, its pretty good.

### CODE REVIEW

File: SQMs/missionTanoa.sqm [hunk 1]
```diff
@@ -51137,5 +51139,208 @@ class Mission
...
+						position[]={5631.0811,3.6714213,10142.964};
+					};
+					side="West";
+					flags=7;
+					class Attributes
+					{
+						isPlayer=1;
+					};
+					id=5775;
+					type="B_soldier_AAA_F";
```
Comment (Jawshy) [inline SQMs/missionTanoa.sqm]: Why is there an [Asst. Missile Specialist](https://community.bistudio.com/wiki/Arma_3_CfgVehicles_WEST#B_soldier_AAA_F)?

**Comment by Jawshy** [comment]
> [...] Further tweaking will be required to remove the giant building which spawns on the hospital_3 marker but apart from that, its pretty good. 

If you remove `hospital_3` from [life_server/init.sqf, line 111](https://github.com/AsYetUntitled/Framework/blob/98d19e4208ec199fbf27ea9f5e5132383eab7327/life_server/init.sqf#L111) then the hospital building won't be placed where the marker is located.

**Comment by Heavybob** [comment]
Asst. Missile Specialist is for uhh.... *deletes commit*

**Comment by Heavybob** [comment]
Will fix it up a little more first.

**Comment by Jawshy** [comment]
Okay, but be sure to keep it to this pull request rather than creating a new one. Thanks.

**Pull request closed without merging**